docs: add posts on outgoing hooks integrations (#707)
devopness/devopness: docs/webhooks/create-outgoing-webhook.md

    ```bash
    curl --request POST \
      --url https://api.devopness.com/users/login \
      --header 'Accept: application/json' \
      --header 'Content-Type: application/json' \
      --data '{
    	"email": "<email>",
    	"password": "<password>"
    }'
    ```
1. From the previous command response, copy the field `access_token`
1. Take note of the `Target URL` (`<target_url>`), `Request Headers` (`<request_headers>`) and `Request Body` (`<request_body>`) fields according to the source provider where the application' source code is hosted, by following the source provider's instructions on the links bellow:
   - Bitbucket: [REST APIs: Create a comment on a pull request](https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/#api-repositories-workspace-repo-slug-pullrequests-pull-request-id-comments-post)
   - Github: [REST API: Create an issue comment](https://docs.github.com/en/rest/issues/comments#create-an-issue-comment)
   - Gitlab: [REST API: Create new issue note](https://docs.gitlab.com/ee/api/notes.html#create-new-issue-note)

> 1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing webhook to comment on the Pull Request using the GitHub API, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`.

Reviewer: ```suggestion
1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing webhook to comment on the Pull Request using your source provider, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`.
```

> 1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing webhook to comment on the Pull Request using your source provider, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`.

Author: ```suggestion
1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing webhook to comment on the Pull Request using the source provider where the application' source code is hosted, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`.
```

    {% raw %}
    ```bash
    curl --request POST \
      --url https://api.devopness.com/hooks/outgoing \
      --header 'Accept: application/json' \
      --header 'Authorization: Bearer <access_token>' \
      --header 'Content-Type: application/json' \
      --data '{

>         "name": "GitHub CD(deploy)",

Reviewer: @thlmenezes same comment of

Author: Post updated

>         "name": "GitHub CD(deploy)",

Author: ```suggestion
        "name": "CD(deploy)",
```

links:
    overview:
    quickstart:
    previous:
    next:
    guides:
    related:
    featured:
---

{% note %}

**NOTE**: Webhooks, for now, are an API only feature; so this post will guide you through the usage of our API to help you create an outgoing webhook.

{% endnote %}

{% note %}

**NOTE**: The examples below use `cURL`, which is a command line utility to be used from a terminal window. If you're not familiar with running commands on a terminal or you simply don't want to use a terminal you could, alternatively, copy the examples and import the `cURL` command line into your favorite API client platform. Here are links with instructions on how to achieve that on some of the most popular API client platforms:

- [Postman](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#importing-with-curl-commands)
- [Insomnia](https://docs.insomnia.rest/insomnia/import-export-data#import-data)

{% endnote %}

> {% note %}
> 
> **NOTE**: Insomnia uses curly braces syntax for environment variables, to avoid errors you need to disable the nunjuncks template feature for the request body. Further instructions at [Insomnia's FAQ](https://docs.insomnia.rest/insomnia/faq#how-can-i-temporarily-disable-nunjucks-template)
> 
> {% endnote %}

Author: ```suggestion
```

1. On your local machine, in a terminal window, submit a request to Devopness API endpoint `POST /hooks/outgoing` to create a outgoing webhook to comment on the Pull Request using the source provider where the application' source code is hosted, replacing `<application_id>`, `<pipeline_id>`, `<target_url>`, `<request_headers>` and `<request_body>`.
    {% raw %}
    ```bash
    curl --request POST \
      --url https://api.devopness.com/hooks/outgoing \
      --header 'Accept: application/json' \
      --header 'Authorization: Bearer <access_token>' \
      --header 'Content-Type: application/json' \
      --data '{
        "name": "CD(deploy)",
        "action_type": "deploy",
        "resource_type": "application",
        "resource_id": <application-id>,
        "target_url": "https:\/\/<target_url>\/{{ action.triggered_from.hook_parsed_variables.pull_request_id }}",
        "settings": {
          "request_headers": [
            {
              "name": "Authorization",
              "value": "Bearer {{ application.source_provider.access_token }}"
            }
            // NOTE: add Request Headers (`<request_headers>`) here
          ],
          "request_body": {

>             // NOTE: review the fields bellow according to Request Body (`<request_body>`)

Author: ```suggestion
            // NOTE: review the fields below according to Request Body (`<request_body>`)
```
